modules/kms-keyring/test/kms_keyring.onencrypt.test.ts

```diff
@@ -81,17 +81,22 @@ describe('KmsKeyring: _onEncrypt', () => {
     expect(edkEncrypt.providerId).to.equal('aws-kms')
     expect(edkEncrypt.providerInfo).to.equal(encryptKmsKey)
 
-    expect(material.keyringTrace).to.have.lengthOf(2)
-    const [traceGenerate, traceEncrypt] = material.keyringTrace
+    expect(material.keyringTrace).to.have.lengthOf(3)
+    const [traceGenerate, traceEncrypt1, traceEncrypt2] = material.keyringTrace
     expect(traceGenerate.keyNamespace).to.equal('aws-kms')
     expect(traceGenerate.keyName).to.equal(generatorKeyId)
     expect(traceGenerate.flags & KeyringTraceFlag.WRAPPING_KEY_GENERATED_DATA_KEY).to.equal(KeyringTraceFlag.WRAPPING_KEY_GENERATED_DATA_KEY)
     expect(traceGenerate.flags & KeyringTraceFlag.WRAPPING_KEY_ENCRYPTED_DATA_KEY).to.equal(KeyringTraceFlag.WRAPPING_KEY_ENCRYPTED_DATA_KEY)
```

Comment:
In the KMS case.  On generate, the encrypted data key is also returned....
It is just so that people can see the relationship between
the generate trace and the encrypt trace.